extension/encoding/awslogsencodingextension/internal/unmarshaler/cloudtraillog/unmarshaler.go

```diff
@@ -108,6 +108,28 @@ type CloudTrailRecord struct {
 	SharedEventID                string         `json:"sharedEventID"`
 }
 
+// logFiles represents log file information in a CloudTrail digest file
+type logFiles struct {
+	S3Bucket        string `json:"s3Bucket"`
+	S3Object        string `json:"s3Object"`
+	NewestEventTime string `json:"newestEventTime"`
+	OldestEventTime string `json:"oldestEventTime"`
+}
+
+// CloudTrailDigest represents a CloudTrail digest file
+type CloudTrailDigest struct {
+	AWSAccountID           string     `json:"awsAccountId"`
+	DigestStartTime        string     `json:"digestStartTime"`
+	DigestEndTime          string     `json:"digestEndTime"`
```

Comment:
We're not translating this. Is that intentional?

Reply:
I left it out given next file will have the `DigestStartTime` which helps to understand which log files belongs to which, but now I think that's not ideal. I will add this in the next commit.